client/src/modules/shopping-list/index.jsx

```diff
@@ -5,7 +5,10 @@ import { withRouter } from 'react-router-dom';
 
 import Toolbar, { ToolbarItem, ToolbarLink } from 'common/components/Toolbar';
 import ProductsContainer from 'modules/shopping-list/components/ProductsContainer';
-import { getList } from 'modules/shopping-list/model/selectors';
+import {
+  getList,
+  getItemsForCurrentList
```

Comment:
I think that name getItems will be enough for this selector (or getListItems)...